images/nested-libvirt/Dockerfile

```diff
@@ -15,6 +15,6 @@ RUN yum install -y \
     gettext \
     google-cloud-sdk \
     openssh-clients && \
-    yum --enablerepo=epel-testing install -y nss_wrapper && \
+    yum install -y nss_wrapper && \
```

Comment:
Might as well update it to `dnf`. :)

Reply:
No `yum` is now symlink for fedora so it will work across platform (be it fedora/centos or RHEL).

Reply:
@sallyom so this change is done because this package is coming from the epel repo instead the epel-testing now. I am hoping initially we did this because specific version we wanted for nss-wrapper not made to the stable repo.

```console
$ yum info nss_wrapper
Failed to set locale, defaulting to C
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: centos4.zswap.net
 * epel: mirror.cogentco.com
 * extras: centos4.zswap.net
 * updates: centos4.zswap.net
Available Packages
Name        : nss_wrapper
Arch        : x86_64
Version     : 1.1.5
Release     : 1.el7
Size        : 37 k
Repo        : epel/x86_64
Summary     : A wrapper for the user, group and hosts NSS API
URL         : https://cwrap.org/
License     : BSD
...
```

Reply:
> I am hoping initially we did this because specific version we wanted for nss-wrapper not made to the stable repo.

I went back through #625 and didn't see any discussion of `--enablerepo=epel-testing`, so I'm fine dropping it here.  We can always restore it later with a commit message about why it is useful, if it turns out that we do, in fact, need it.